### PR TITLE
OBSDOCS-542: Update logging pod names

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-542

Link to docs preview:
- https://67890--docspreview.netlify.app/openshift-enterprise/latest/logging/troubleshooting/cluster-logging-cluster-status
- https://67890--docspreview.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-moving-nodes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Comment by QiaolingTang** [comment]
LGTM.

File: modules/infrastructure-moving-logging.adoc [hunk 1]
```diff
@@ -184,12 +178,12 @@ cluster-logging-operator-84d98649c4-zb9g7       1/1     Running       0
 elasticsearch-cdm-hwv01pf7-1-56588f554f-kpmlg   2/2     Running       0          28m
 elasticsearch-cdm-hwv01pf7-2-84c877d75d-75wqj   2/2     Running       0          28m
 elasticsearch-cdm-hwv01pf7-3-f5d95b87b-4nx78    2/2     Running       0          28m
-fluentd-42dzz                                   1/1     Running       0          28m
-fluentd-d74rq                                   1/1     Running       0          28m
-fluentd-m5vr9                                   1/1     Running       0          28m
-fluentd-nkxl7                                   1/1     Running       0          28m
-fluentd-pdvqb                                   1/1     Running       0          28m
-fluentd-tflh6                                   1/1     Running       0          28m
+collector-42dzz                                   1/1     Running       0          28m
```
Comment (ShaunaDiaz) [inline modules/infrastructure-moving-logging.adoc:181]:
```suggestion
collector-42dzz                                 1/1     Running       0          28m
```
Looks like these have a few extra spaces and aren't aligning starting with the pod count.

File: modules/infrastructure-moving-logging.adoc [hunk 2]
```diff
@@ -223,11 +217,11 @@ cluster-logging-operator-84d98649c4-zb9g7       1/1     Running   0          30m
 elasticsearch-cdm-hwv01pf7-1-56588f554f-kpmlg   2/2     Running   0          29m
 elasticsearch-cdm-hwv01pf7-2-84c877d75d-75wqj   2/2     Running   0          29m
 elasticsearch-cdm-hwv01pf7-3-f5d95b87b-4nx78    2/2     Running   0          29m
-fluentd-42dzz                                   1/1     Running   0          29m
-fluentd-d74rq                                   1/1     Running   0          29m
-fluentd-m5vr9                                   1/1     Running   0          29m
-fluentd-nkxl7                                   1/1     Running   0          29m
-fluentd-pdvqb                                   1/1     Running   0          29m
-fluentd-tflh6                                   1/1     Running   0          29m
+collector-42dzz                                   1/1     Running   0          29m
```
Comment (ShaunaDiaz) [inline modules/infrastructure-moving-logging.adoc:220]: Same here, these bumped out a handful of spaces.

**Review by ShaunaDiaz — APPROVED** [review]
Looks great! Just a few alignment issues in the sample outputs for the moving logging assembly.

**Review by ShaunaDiaz — APPROVED** [review]
Looks great! Just a few alignment issues in the sample outputs for the moving logging assembly.

**Pull request merged**